.github/workflows/base-component-descriptor.yaml

```diff
@@ -26,6 +26,23 @@ on:
         required: false
         type: string
         default: SAP SE
+      component-references:
+        required: false
+        type: string
+        description: |
+          relative path (to repo root) to YAML/Json file containing definition of
```

Comment:
two nits:
- JSON-documents are _always_ also YAML-documents (so there is no actually added information by mentioning JSON explicitly)
- JSON is an acronym (JavaScript Object Notation), so by convention it should also be set in uppercase